articles/machine-learning/how-to-auto-train-image-models.md

```diff
@@ -547,7 +547,7 @@ image_object_detection_job = automl.image_object_detection(
     training_data=my_training_data_input,
     validation_data=my_validation_data_input,
     target_column_name="label",
-    primary_metric="mean_average_precision",
```

Comment:
does this need to be updated in the new batch scoring notebook as well that Shubham added?

Reply:
oops.. my bad.. this is doc..